src/base/settingsstorage.cpp

```diff
@@ -65,12 +65,12 @@ namespace
     QString mapKey(const QString &key)
     {
         static const MappingTable keyMapping = {
-
```

Comment:
Why not, it's a empty line which doesn't mean anything except taking vertical spaces.